src/app.py

```diff
@@ -65,6 +65,12 @@ def bootstrap(_return=0):
         try:
             master_host = get_or_raise('MASTER_HOST')
             master_url = 'http://{master}:8089'.format(master=master_host)
+            total_slaves = int(os.getenv('TOTAL_SLAVES')) if os.getenv('TOTAL_SLAVES') else int(
+                os.getenv('SLAVE_MUL', multiprocessing.cpu_count()))
+            # Default time duration to wait all slaves to be connected is 1 minutes / 60 seconds
+            SLAVES_WAITING_TIME = float(os.getenv('SLAVES_CHECK_TIMEOUT', 60))
```

Comment:
Could you please also rename python vars so they match the env vars and make them lower-case, just like the total_slaves? It will look more consistent.